spring-boot-test-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -98,3 +98,13 @@ org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListen
 org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener,\
 org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener,\
 org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener
+
+# AutoConfigureEmbeddedTestMongod auto-configuration imports
+org.springframework.boot.test.autoconfigure.data.mongo.AutoConfigureEmbeddedTestMongod=\
+org.springframework.boot.test.autoconfigure.data.mongo.EmbeddedTestMongodAutoConfiguration
+
+# AutoConfigureDataMongo auto-configuration imports
+org.springframework.boot.test.autoconfigure.data.mongo.AutoConfigureDataMongo=\
+org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration,\
+org.springframework.boot.autoconfigure.data.mongo.MongoDataAutoConfiguration,\
```

Comment:
To be clear, I'd advise to include `org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration` here.